alerts/machine-config-operator/ExtremelyHighIndividualControlPlaneMemory.md

```diff
@@ -0,0 +1,27 @@
+# ExtremelyHighIndividualControlPlaneMemory
+
+## Meaning
+
+This alert fires when the memory use by an instance within control plane
+nodes exceeds 90% of total memory available on that node.  
+This is a critical level alert.
+
+## Impact
+
+At such very low free memory, the cluster becomes unstable you can expect slow
+responses from kube-apiserver or failing requests specially from etcd.
+Moreover, OOM kill is expected, which negatively influences the pod scheduling.
+
+## Diagnosis
+
+To fix this, increase memory of the affected node of control plane nodes.
```

Comment:
This would fit better into the `Mitigation` section I think.
For `Diagnosis` I'd expect some information on how to gain more insight into the issue, like which components use a lot of memory, how much scaling up would be needed to meet demand (to inform the `Mitigation` steps) or similar steps.

Reply:
+1, this section should cover steps which user can take to figure out possible reason that lead to high memory usage.